playbooks/roles/pipeline/tasks/setup_pipelines.yml

```diff
@@ -29,14 +31,15 @@
   shell: "{{ oc_bin }} get buildconfig | grep 'JenkinsPipeline' | awk '{print $1}' | egrep '^{{ item.path.split('/')[-2] }}$'"
   register: "pipeline_name_check"
   with_items: "{{ all_pipelines }}"
+  when: pipelines is defined
   ignore_errors: yes
 
 - name: "Replace pipeline buildconfigs"
-  shell: "{{ oc_bin }} replace -f {{ item.item.path }}"
+  command: "{{ oc_bin }} replace -f {{ item.item.path }}"
```

Comment:
I rather use shell in this case since command may have issues in that it is just a command execution

Reply:
I agree but the ansible-lint show as wrong. I guess that if the command has "|", "&&" or ";;" make sense, but a single command with your parameters/arguments we could be following the ansible-lint to get a compliance with this tool.

Reply:
There are plenty of places besides these two that we use shell so that means we would have to change them all and it will not work I tested it.

Reply:
I would like see a quality check here, if the ansible-lint doesn't work we should send a patch or report to them or find a solution to validade our yaml playbooks because at moment have more than 100 issues opened from ansible-lint. https://gist.github.com/firemanxbr/9e59115413939a3e4032932dc699a3d6

Reply:
It isn't a quality issue it is a linter.  We have to use a command or shell there is no other alternative.